doc-content/enterprise-only/bpmn/bpmn-tasks-overview-con.adoc

```diff
@@ -30,14 +30,14 @@ In additions, the BPMN2 specification provides the ability to create custom task
 
 
 .Business rule task 
-A business rule task defines a set of rules that are evaluated and fired on task execution. Any rule defined as part of the rule flow group in a rule resource is fired.
+A business rule task defines a way to make a decision either through a DMN model or a rule flow group.
 
 image::BPMN2/bpmn-business-rule-task.png[]
 
 
-When a process reaches a business rule task, the engine starts executing the rules with the defined rule flow group.
-When there are no more active rules with the rule flow group, the execution continues to the next element.
-During the rule flow group execution, new activations belonging to the active rule flow group can be added to the agenda because these acativations are changed by other rules.
+When a process reaches a business rule task defined by a DMN model, the engine executes the DMN model decision with the inputs provided.
+
+When a process reaches a business rule task defined by a rule flow group, the engine begins executing the rules in the defined rule flow group. When there are no more active rules in the rule flow group, the execution continues to the next element. During the rule flow group execution, new activations belonging to the active rule flow group can be added to the agenda because these activations are changed by other rules.
```

Comment:
We should avoid hard-coding "engine" and specify which engine with the variable, either `{PROCESS_ENGINE}` or `{DECISION_ENGINE}`. For DM, it's always the latter. For PAM, it's one or the other. PAM has both engines and it's usually easy to tell which one is being used, but in this case it's tough because it's about rule execution within a process. My guess, based on the wording, is that it's `{PROCESS_ENGINE}`, because you're talking about execution moving from one rule-based execution to the next in the process. Each individual rule execution, though, is handled by the decision engine. I'd just go with process engine, but if you want to leave it generically hard-coded, that's up to you.

Reply:
It's the process engine.

Reply:
and it's process engine for DM too ... because in DM you can model rule flows in Process Designer which use business rule tasks.